test/testUtils.ts

```diff
@@ -1,28 +1,38 @@
 "use strict";
+import * as execa from "execa";
```

Comment:
This file has to stay a .js. It's used to test bin cases where we have only .js files. I don't think it's a good thing to mix ts and js.

Bin cases tests real world examples where js files are consumed by webpack. I don't see any point to move them to ts

Reply:
I wasn't aware of that part. we should update the documentation!